examples/experimental/i3s-17-and-debug/package.json

```diff
@@ -44,7 +47,9 @@
     "babel-loader": "^8.0.5",
     "babel-plugin-inline-import": "^3.0.0",
     "babel-polyfill": "^6.26.0",
+    "css-loader": "^5.1.2",
```

Comment:
Since we are using styled components, can we avoid these webpack loaders? The example setup should be as simple as possible.

Regardless, these should be `devDependencies`.